### PR TITLE
Update to use new rspec-puppet-facts version

### PR DESCRIPTION
**Pull request by stbenjam**
- [ ] https://github.com/Katello/puppet-candlepin/pull/51
- [x] https://github.com/Katello/puppet-capsule/pull/95
- [x] https://github.com/Katello/puppet-certs/pull/106
- [x] https://github.com/Katello/puppet-common/pull/13
- [x] https://github.com/Katello/puppet-crane/pull/13
- [x] https://github.com/Katello/puppet-katello/pull/144
- [x] https://github.com/Katello/puppet-katello_devel/pull/88

### CODE REVIEW

**Comment by ehelms** [comment]
ACK

**Comment by stbenjam** [comment]
Best to hold on reviewing this or the linked PR's, I'm still working on this, it may need some changes

File: moduleroot/spec/spec_helper.rb [hunk 1]
```diff
@@ -9,6 +9,11 @@
 require 'rspec-puppet-facts'
 include RspecPuppetFacts
 
+add_custom_fact :concat_basedir, '/tmp'
```
Comment (ekohl) [inline moduleroot/spec/spec_helper.rb]: Would it be useful to have a comment where each fact usually comes from? Like puppetlabs-concat in this case

Comment (stbenjam) [reply moduleroot/spec/spec_helper.rb]: Yea, probably

Comment (stbenjam) [reply moduleroot/spec/spec_helper.rb]: Updated

**Comment by stbenjam** [comment]
@ekohl Would you mind looking again?

**Review by ekohl — APPROVED** [review]

**Comment by stbenjam** [comment]
Thanks everyone!

**Pull request merged**